plugins/org.yakindu.base.xtext.utils.jface/src/org/yakindu/base/xtext/utils/jface/viewers/util/ActiveEditorTracker.java

```diff
@@ -150,6 +153,15 @@ public static IProject getLastActiveEditorProject() {
 		if (editorInput instanceof IFileEditorInput) {
 			final IFileEditorInput input = (IFileEditorInput) editorInput;
 			return input.getFile().getProject();
+		} else if (editorInput instanceof URIEditorInput) {
+			final URI uri = ((URIEditorInput) editorInput).getURI();
+			if (uri.isPlatformResource()) {
+				final String segment = uri.segment(1);
```

Comment:
Not sure if segment(1) is always the project name. This is how we map from an URI to an IFile in other places

if(uri.isPlatformResource()) {
	String platformString = uri.toPlatformString(true);
	return ResourcesPlugin.getWorkspace().getRoot().findMember(platformString).getProject();
}